# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipconfig/hotfix.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ipconfig/netdev.h.vanilla	2019-09-17 15:55:39.308000000 +0200
+++ ipconfig/netdev.h	2019-09-17 15:55:49.672000000 +0200
@@ -8,6 +8,8 @@
 #include <sys/utsname.h>
 #include <net/if.h>
 
+#define SYS_NMLN  512
+
 #define BPLEN		40
 
 struct netdev {
--- ipconfig/packet.c.vanilla	2019-09-17 15:58:26.960000000 +0200
+++ ipconfig/packet.c	2019-09-17 15:58:31.964000000 +0200
@@ -1,3 +1,4 @@
+#define _GNU_SOURCE
 /*
  * ipconfig/packet.c
  *
